test/src/test/java/jenkins/model/JenkinsTest.java

```diff
@@ -838,4 +840,35 @@ public String getUrlName() {
             return "login123";
         }
     }
+
+    @Test
+    public void checkInitialView() {
+        assertTrue(CheckInitialViewExtension.hasPrimaryView);
+    }
+
+    @TestExtension(value = "checkInitialView")
```

Comment:
By the way I think this has no actual effect—this `@Initializer` will be run regardless of which test is running (all of `jenkins-test`).